redfish-core/lib/pcie.hpp

```diff
@@ -238,12 +303,22 @@ inline void requestRoutesSystemPCIeFunctionCollection(App& app)
                     asyncResp->res.jsonValue["PCIeFunctions@odata.count"] =
                         pcieFunctionList.size();
                 };
-                std::string escapedPath = std::string(pciePath) + "/" + device;
-                dbus::utility::escapePathForDbus(escapedPath);
-                crow::connections::systemBus->async_method_call(
-                    std::move(getPCIeDeviceCallback), pcieService, escapedPath,
-                    "org.freedesktop.DBus.Properties", "GetAll",
-                    pcieDeviceInterface);
+
```

Comment:
I'm curious why the path escaping was removed?

Reply:
ok.  I probably would have left the escaping in since it doesn't hurt anything and provides broader compatibility with applications like entity-manager....but we probably won't be supporting that with this fork of bmcweb so I guess it probably doesn't matter.